examples/checkout/create_payment_request.rb

```diff
@@ -14,8 +14,7 @@
 
 # if you don't want use the application config, can give your credentials object to payment request
```

Comment:
I don't think so.

It is not talking about (un)comment lines, it is just saying: "you can use there or here".

Suggestions are welcome. :)